tests/disk/test_cli.py

```diff
@@ -0,0 +1,573 @@
+    @mock.patch(
+        "subprocess.check_output", return_value=b"/dev/pretend/luks-id-123456\n"
+    )
+    @mock.patch("subprocess.check_call", return_value=0)
+    def test_mount_volume_already_mounted(
+        self, mocked_output, mocked_call, mocked_path
+    ):
+        md = Volume(
+            device_name=_DEFAULT_USB_DEVICE_ONE_PART,
+            mapped_name=_PRETEND_LUKS_ID,
+            encryption=EncryptionScheme.LUKS,
+        )
+        result = self.cli.mount_volume(md)
+        assert result.mountpoint == "/dev/pretend/luks-id-123456"
+
+    @mock.patch("os.path.exists", return_value=True)
+    @mock.patch("subprocess.check_output", return_value=b"\n")
+    @mock.patch("subprocess.check_call", return_value=0)
+    def test_mount_volume_mkdir(self, mocked_output, mocked_subprocess, mocked_path):
+        md = Volume(
+            device_name=_DEFAULT_USB_DEVICE_ONE_PART,
+            mapped_name=_PRETEND_LUKS_ID,
+            encryption=EncryptionScheme.LUKS,
+        )
+        assert self.cli.mount_volume(md).mapped_name == _PRETEND_LUKS_ID
+
+    @mock.patch("subprocess.check_output", return_value=b"\n")
+    @mock.patch(
+        "subprocess.check_call",
+        side_effect=subprocess.CalledProcessError(1, "check_call"),
+    )
+    def test_mount_volume_error(self, mocked_subprocess, mocked_output):
+        md = Volume(
+            device_name=_DEFAULT_USB_DEVICE_ONE_PART,
+            mapped_name=_PRETEND_LUKS_ID,
+            encryption=EncryptionScheme.LUKS,
+        )
+
+        with pytest.raises(ExportException) as ex:
+            self.cli.mount_volume(md)
+
+        assert ex.value.sdstatus is Status.ERROR_MOUNT
+
+    @mock.patch("os.path.exists", return_value=False)
+    @mock.patch(
+        "subprocess.check_call",
+        side_effect=subprocess.CalledProcessError(1, "check_call"),
+    )
+    def test_mount_at_mountpoint_mkdir_error(self, mocked_subprocess, mocked_path):
+        md = Volume(
+            device_name=_DEFAULT_USB_DEVICE_ONE_PART,
+            mapped_name=_PRETEND_LUKS_ID,
+            encryption=EncryptionScheme.LUKS,
+        )
+
+        with pytest.raises(ExportException) as ex:
+            volume = self.cli._mount_at_mountpoint(md, self.cli._DEFAULT_MOUNTPOINT)
+            assert not volume.writable
+
+        assert ex.value.sdstatus is Status.ERROR_MOUNT
+
+    @mock.patch("os.path.exists", return_value=True)
+    @mock.patch(
+        "subprocess.check_call",
+        side_effect=subprocess.CalledProcessError(1, "check_call"),
+    )
+    def test_mount_at_mountpoint_mounting_error(self, mocked_subprocess, mocked_path):
+        md = Volume(
+            device_name=_DEFAULT_USB_DEVICE_ONE_PART,
+            mapped_name=_PRETEND_LUKS_ID,
+            encryption=EncryptionScheme.LUKS,
+        )
+
+        with pytest.raises(ExportException) as ex:
+            volume = self.cli._mount_at_mountpoint(md, self.cli._DEFAULT_MOUNTPOINT)
```

Comment:
Same questions, notes, here about testing this private method: isn't this meaningfully covered by the `self.cli.mount_volume(...)` tests?

Reply:
Probably falls into the same category as https://github.com/freedomofpress/securedrop-export/pull/105#discussion_r1067553656